### ISSUE TITLE
Question: Static Resources?

### ISSUE THREAD

**Issue by claytonk** [jpdevries/hits#15]
Is there anyway this can be made to work with static resources?

**Comment by jpdevries** [comment]
Hm, well I don't see why it wouldn't work with static resources. I assume it isn't for you though? Are you just trying to log a 'visit' or 'hit' when a user visits a resource that happens to be static? Is the snippet being called uncached?

**Comment by claytonk** [comment]
No, doesn’t seem to be recording hits on static resources. I’m calling the snippet uncached from inside my template <head>, I assumed it wasn’t working on static resources simply because they don’t really use the template code as a normal resource would — but maybe I’m wrong on that. I assumed I would need to work it in as a plugin tied to a system event, but if you can correct me on that I’d welcome your insights.

Clayton Kinney
316 Creative
Office: 410.747.8052
Mobile: 301.646.1521
